website/docs/plugin/framework/migrating/attributes-blocks/attribute-schema.mdx

```diff
@@ -0,0 +1,129 @@
+---
+page_title: 'Attribute Schema: Migrating from SDKv2 to the Framework'
+description: >-
+  Migrate attributes from SDKv2 to the plugin Framework
+---
+
+# Attribute Schema
+
+Attributes define how users can configure values for your Terraform provider, resources, and data sources. Refer to
+[Schemas - Attributes](/plugin/framework/schemas#attributes) in the Framework documentation for details.
+
+This page explains how to migrate an attribute from SDKv2 to the plugin Framework.
+
+## SDKv2
```

Comment:
Have [installed](https://github.com/igorshubovych/markdownlint-cli) and run ` markdownlint --disable MD013 -- ./website/docs/plugin/framework/migrating/*`.

I'm ignoring the `MD013/line-length Line length` notifications.

In terms of the remaining notifications, I'm interested to hear feedback from @robin-norwood and @laurapacilio as I thought this was intentional but happy to fix however suits:
```go
markdownlint --disable MD013 -- ./website/docs/plugin/framework/migrating/*
./website/docs/plugin/framework/migrating/data-sources.mdx:108 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### SDKv2"]
./website/docs/plugin/framework/migrating/data-sources.mdx:141 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### Framework"]
./website/docs/plugin/framework/migrating/provider.mdx:111 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### SDKv2"]
./website/docs/plugin/framework/migrating/provider.mdx:139 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### Framework"]
./website/docs/plugin/framework/migrating/provider.mdx:209 MD036/no-emphasis-as-heading/no-emphasis-as-header Emphasis used instead of a heading [Context: "SDKv2"]
./website/docs/plugin/framework/migrating/provider.mdx:234 MD036/no-emphasis-as-heading/no-emphasis-as-header Emphasis used instead of a heading [Context: "Framework"]
./website/docs/plugin/framework/migrating/provider.mdx:279 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### SDKv2"]
./website/docs/plugin/framework/migrating/provider.mdx:295 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### Framework"]
./website/docs/plugin/framework/migrating/provider.mdx:315 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### Migration Notes"]
./website/docs/plugin/framework/migrating/provider.mdx:323 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### Example"]
./website/docs/plugin/framework/migrating/provider.mdx:337 MD036/no-emphasis-as-heading/no-emphasis-as-header Emphasis used instead of a heading [Context: "SDKv2"]
./website/docs/plugin/framework/migrating/provider.mdx:360 MD036/no-emphasis-as-heading/no-emphasis-as-header Emphasis used instead of a heading [Context: "Framework"]
./website/docs/plugin/framework/migrating/testing.mdx:109 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### Example"]
./website/docs/plugin/framework/migrating/testing.mdx:119 MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
./website/docs/plugin/framework/migrating/testing.mdx:120 MD036/no-emphasis-as-heading/no-emphasis-as-header Emphasis used instead of a heading [Context: "SDKv2"]
./website/docs/plugin/framework/migrating/testing.mdx:147 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### Framework"]
```

Reply:
Forgive me - is the problem that we have duplicate headings? My opinion is that they are needed here, but let me know if you disagree!

Reply:
The 2 things that are being flagged by the linter are:

- Duplicate headings (e.g., where we have ### SDKv2 multiple times).
- Emphasis used instead of heading (e.g., **SDKv2**) rather than #### SDKv2, such as in the ### Example sections.

Reply:
Following the removal of `**SDKv2**` and `**Framework**`, we now have the following:
```
terraform-plugin-framework % markdownlint --disable MD013 -- ./website/docs/plugin/framework/migrating/*
./website/docs/plugin/framework/migrating/testing.mdx:108 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### Example"]
./website/docs/plugin/framework/migrating/testing.mdx:118 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "#### SDKv2"]
./website/docs/plugin/framework/migrating/testing.mdx:144 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "#### Framework"]
```